source/codegen/metadata/nixnet/functions.py

```diff
@@ -0,0 +1,1539 @@
+functions = {
+    'Blink': {
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'interfaceRef',
+                'type': 'nxSessionRef_t'
+            },
+            {
+                'direction': 'in',
+                'enum': 'BlinkMode',
+                'name': 'modifier',
+                'type': 'u32'
+            }
+        ],
+        'returns': 'nxStatus_t'
+    },
+    'Clear': {
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'sessionRef',
```

Comment:
Same as my comment above. I *think* the best name for all of these is to use `session` for the name of everything that is an input/output of `nxSessionRef_t`.

You can use `grpc_name` for that and leave `name` the same (either way is fine, `grpc_name` was more important for RFmx, so that's my current recommendation).

This doesn't need to be in this change. But I'd recommend figuring it out soon. Since all existing code would need to update for it.